### PR TITLE
Add File Component

### PR DESCRIPTION
**Pull request by fspoettel**

### CODE REVIEW

File: src/components/File/File.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    hideTitle,
+    image,
+    isLoading,
+    ratio,
+    title,
+    ...rest
+  } = props;
+
+  const renderActions = () => {
+    if (!hideTitle) {
```
Comment (philippfrank) [inline src/components/File/File.js]: I don't get this. Seems it should be `if (hideTitle)`?

File: src/components/File/File.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    actions,
+    className,
+    externalIcon,
+    fileType,
+    hideTitle,
+    image,
+    isLoading,
+    ratio,
+    title,
+    ...rest
```
Comment (philippfrank) [inline src/components/File/File.js]: Although the linter allows this, can you add a comma to be consistent.

Comment (fspoettel) [reply src/components/File/File.js]: no: https://github.com/eslint/eslint/issues/3627

File: src/components/File/File.js (hunk 1 above)
Comment (fspoettel) [reply src/components/File/File.js]: the logic is correct, I'll switch the statements tho. That's what I get for renaming `showTitle` to `hideTitle` later in dev

**Pull request merged**